### PR TITLE
feat(llms): default values for default_llm_factories

### PR DESCRIPTION
**Pull request by kdziedzic68**

### CODE REVIEW

**Review by micpst — APPROVED** [review]

File: packages/ragbits-core/src/ragbits/core/llms/factory.py [hunk 1]
```diff
@@ -75,3 +72,14 @@ def simple_litellm_vision_factory() -> LLM:
...
     return LiteLLM(model_name="gpt-4o-mini")
+
+
+def simple_litellm_structured_output_factory() -> LLM:
+    """
+    A basic LLM factory that creates an LiteLLM instance with the structured output.
+    Functionality supported
+
+    Returns:
+        LLM: An instance of the LiteLLM.
```
Comment (ludwiktrammer) [inline packages/ragbits-core/src/ragbits/core/llms/factory.py:83]:
```suggestion
        LLM: An instance of the LiteLLM class.
```

**Comment by ludwiktrammer** [comment]
It would make sense to also remove code that defines and uses `DEFAULT_LLM_IMAGE_DESCRIPTION_MODEL` in `packages/ragbits-document-search/src/ragbits/document_search/ingestion/providers/unstructured/images.py` - after this PR, it no longer makes sense.

File: packages/ragbits-core/src/ragbits/core/llms/factory.py [hunk 2]
```diff
@@ -75,3 +72,14 @@ def simple_litellm_vision_factory() -> LLM:
         LLM: An instance of the LiteLLM.
     """
     return LiteLLM(model_name="gpt-4o-mini")
+
+
+def simple_litellm_structured_output_factory() -> LLM:
+    """
+    A basic LLM factory that creates an LiteLLM instance with the structured output.
+    Functionality supported
```
[the remark this answers is not in the document]
Comment (kdziedzic68) [reply packages/ragbits-core/src/ragbits/core/llms/factory.py:80]: done

[the remark this answers is not in the document]
Comment (kdziedzic68) [reply packages/ragbits-core/src/ragbits/core/llms/factory.py]: ok - done forgot to push something previously

**Review by ludwiktrammer — APPROVED** [review]

**Pull request merged**